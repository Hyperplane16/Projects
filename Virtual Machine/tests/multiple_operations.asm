FUNC LABEL 0
    MOV REG 0 VAL 5
    MOV REG 0 VAL 150
    CAL VAL 1
    ADD REG 0 REG 1
    PRINT REG 0
    NOT REG 0
    PRINT REG 0
    RET
FUNC LABEL 1
    MOV REG 1 VAL 25
    PRINT REG 1
    RET
