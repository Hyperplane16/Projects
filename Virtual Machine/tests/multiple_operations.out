25
175
80
0